app/main.py

```diff
@@ -1,5 +1,72 @@
 class Cargo:
-    def __init__(self, weight):
+    def __init__(self, weight: int) -> None:
         self.weight = weight
 
-# write your code here
+
+class BaseRobot:
+    def __init__(self,
+                 name: str,
+                 weight: int,
+                 coords: list = None
+                 ) -> None:
```

Comment:
```suggestion

    def __init__(
             self, 
             name: str, 
             weight: int,
             coords: list = None
     ) -> None:
```
It would be great if you provide this style with parentheses all over the project